crates/bevy_ecs/src/reflect.rs

```diff
@@ -123,6 +124,90 @@ impl<C: Component + Reflect + FromWorld> FromType<C> for ReflectComponent {
     }
 }
 
+#[derive(Clone)]
+pub struct ReflectResource {
```

Comment:
Basic doc strings please.

Reply:
But `ReflectComponent` also doesn't have a doc string. Should I add it anyway?

Reply:
Yes please. I'd be fine if you want to add doc strings to `ReflectComponent` in this PR too.

Always nice to leave the code base a little better than you found it.